File: java/SR-RBD/src/ddb/tpc/LessTableVersionException.java
package ddb.tpc;

public class LessTableVersionException extends Exception {
	
	public LessTableVersionException(String tableName, int coordinatorTableVersion, int localTableVersion) {
		super("Less table version " + tableName + " coordinator: " + coordinatorTableVersion + " local:" + localTableVersion );
	}

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

}

File: C#/RBD/RBD/TPC/LessTableVersionException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RBD.TPC
{
    public class LessTableVersionException : Exception
    {
        public LessTableVersionException(String tableName, int coordinatorTableVersion, int localTableVersion)
            : base("Less table version " + tableName + " coordinator: " + coordinatorTableVersion + " local:" + localTableVersion)
        {
        }
    }
}
